migrations/gender.txt

```diff
@@ -1,6 +1,6 @@
 Description: Update gender constraints for snap-user table.
```

Comment:
Please, change the name of the file to include a timestamp on its name... so when you have more than one migration, you can list it in order... for example:
2018-07-25-15-45-23_create_type_gender.txt

Reply:
@dbalseiro I was wondering if that is not the same as specifing dependency on each migration, like in `gender.txt` is `Depends: initial`

Reply:
No... it's not

Reply:
here is how migrations looks like in a project:
![screenshot from 2018-07-26 09-39-47](https://user-images.githubusercontent.com/2053849/43269293-29afcc7c-90b8-11e8-8836-0ba03b8e8a46.png)